pkg/deploy/registry/deploylog/rest.go

```diff
@@ -180,6 +186,31 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
 	}, nil
 }
 
+// waitForExistingDeployment will use the timeout to wait for a deployment to appear.
+func (r *REST) waitForExistingDeployment(namespace, name string) (*kapi.ReplicationController, error) {
+	var (
+		target *kapi.ReplicationController
+		err    error
+	)
+
+	condition := func() (bool, error) {
```

Comment:
@smarterclayton i think what you are asking is reasonable but it's different from what we are doing at the moment ie. get the logs out of a specific deployer pod or application pod. Feels like a separate issue, WDYT?